metamorph/plugins/morph_resultsdb.py

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/python
+import argparse
+import logging
+import logging.config
+import json
+import time
+import os
+import requests
+
+from metamorph.lib.logging_conf import setup_logging
+
+
+class ResultsDBApiException(Exception):
+    pass
+
+
+class ResultsDBApi(object):
+    """
+    Class to communicate and process data with resultsDB
+    """
+    TIMEOUT_LIMIT = 7200  # Wait 2 hours maximally
+    RESULTSDB_LIMITER = 10
+    MINUTE = 60  # 60 seconds
```

Comment:
I generally don't like having magic numbers in my code.
@gsr-shanks  What do you think?